emcc.py

```diff
@@ -2791,6 +2791,59 @@ def in_directory(root, child):
   return False
 
 
+
+def parse_value(text):
+  def _parse_string_value(text):
+    first = text[0]
+    if first == "'" or first == '"':
+      assert text[-1] == text[0] and len(text) > 1, 'unclosed opened quoted string. expected final character to be "%s" and length to be greater than 1 in "%s"' % (text[0],text)
+      return text[1:-1]
+    else:
+      return text
+
+  def _parse_string_list_inner(text, sep): # Part of parse_value
+    values = text.split(sep)
+    result = []
+    index = 0
+    while True:
+      current = values[index].lstrip() # Cannot remove all at start/end because that could affect a string with a comma
+      first = current[0]
```

Comment:
this may throw if `current` is the empty string, we should probably throw an error with a message instead